mlflow/store/cli.py

```diff
@@ -0,0 +1,101 @@
+from mlflow.utils.logging_utils import eprint
+
+import click
+
+from mlflow.tracking import _get_store
+from mlflow.store.artifact_repo import ArtifactRepository
+from mlflow.utils.proto_json_utils import message_to_json
+
+
+@click.group("artifacts")
+def commands():
+    """Upload, list, and download artifacts from an MLflow artifact repository."""
+    pass
+
+
+@commands.command("log-artifact",
+                  help="Logs a local file as an artifact of a run, optionally within a " +
+                       "run-specific artifact path. Run artifacts can be organized into " +
+                       "directories, so you can place the artifact in a directory this way.")
+@click.option("--local-file", "-l", required=True,
```

Comment:
I'd call this local-path since it can be a directory too.

Reply:
Hmm, actually it can't be a directory in GCS, DBFS, S3, or SFTP (only FileStore).